projects/cloudflare/cfssl/Makefile

```diff
@@ -0,0 +1,37 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.14"
+
+REPO=cfssl
+REPO_OWNER=cloudflare
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base
```

Comment:
what do they base off upstream?

Reply:
Actually they have 2 Docker files but looking at their make target they seem to be building from this file
https://github.com/cloudflare/cfssl/blob/master/Dockerfile
Which just uses golang